coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = result.__dict__
```

Comment:
you could also use `dir` or `vars`, they are maybe better as these are built-ins^^